website/pages/docs/plugins/destinations/clickhouse/configuration.mdx

```diff
@@ -0,0 +1,67 @@
+# ClickHouse destination plugin configuration reference
```

Comment:
Let's move the contents of this file to the overview page and delete this one, there's no need to split them at this stage